extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -1068,7 +1070,7 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
       const transactionResponse = responseQueue.pop();
 
       if (typeof transactionResponse === "function") {
-        transactionResponse(response);
+        transactionResponse(response, sourceKeys.publicKey());
```

Comment:
now the signing methods also return the address that signed them - so adding this as a second param